.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "trailingComma": "es5",
+  "semi": false,
```

Comment:
😢

Reply:
Copied from gatsbyjs/gatsby! I have no preference whatsoever, happy to change it.

Reply:
I'm just kidding. We should keep it consistent, it's just not _my_ taste. 😄